source/fundamentals/collations.txt

```diff
@@ -4,5 +4,302 @@ Collations
+
+   To learn how to insert documents, see :ref:`golang-insert-guide`.
+   
+After inserting these documents, you can use an operation with the default collation
+on the ``books`` collection specified in the :ref:`golang-collection-collation`
+section. The following example uses the ``Find()`` method to return all documents with
+a ``name`` value that alphabetically precedes ``"Infinite Jest"``:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+      filter := bson.D{{"name", bson.D{{"$lt", "Infinite Jest"}}}}
+      cursor, err := coll.Find(context.TODO(), filter)
+
+      var results []bson.D
+      if err = cursor.All(context.TODO(), &results); err != nil {
+         panic(err)
+      }
+      for _, result := range results {
+         fmt.Println(result)
+      }
+
+   .. output::
+      :language: none
+      :visible: false
+
+      [{name Ça} {author King} {length 1138}]
+      [{name Cryptonomicon} {author Stephenson} {length 918}]
+      [{name Emma} {author Austen} {length 474}]
+
+Without specifying a default ``books`` collation, the ``Find()`` method would follow default
+binary collation rules to determine the ``name`` values that precede ``"Infinite Jest"``. These
+rules place words beginning with "Ç" after those beginning with "I". The output would resemble
+the following:
+
+.. code-block:: json
+   :copyable: false
+
+   [{name Cryptonomicon} {author Stephenson} {length 918}]
+   [{name Emma} {author Austen} {length 474}]
+
+See :ref:`golang-retrieve` for help with using the ``Find()`` method.
+
+Set a Collation on an Index 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+After creating the ``books`` collection and specifying a collation, you cannot set a new default 
+collation directly. However, you can create an index with a new collation. 
+
+The following example uses the ``CreateOne()`` method to create an ascending index on the ``name``
+field and specifies a new collation with an ``"en_US"`` locale:
+
+.. io-code-block::
+   :copyable: true
+
+   .. input::
+      :language: go
+
+       myCollation := &options.Collation{Locale: "en_US"}
+       opts := options.Index().SetCollation(myCollation)
+
+       indexModel := mongo.IndexModel{
+         Keys:    bson.D{{"name", 1}},
+         Options: opts,
+       }
+
+       name, err := coll.Indexes().CreateOne(context.TODO(), indexModel)
+       if err != nil {
+         panic(err)
+       }
+       fmt.Println("Name of Index Created: " + name)
+
+   .. output::
+      :language: none
+      :visible: false
+
+       Name of Index Created: name_1
+
+
+Set a Collation on an Operation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Operations that read, update, and delete documents from a collection can use collations. 
+
+The following example uses the ``Find()`` method to return documents with ``length`` 
+values greater than ``"1000"``. The ``NumericOrdering`` collation field has a value of
+``true`` to ensure numeric comparison, rather than string comparison:
```

Comment:
Issue: I believe the strings (that contain numeric values) are still compared, so I'm not sure it is correct to say "rather than string comparison". I think this should read "numerical order" rather than "alphabetical order" (language from [wikipedia](https://en.wikipedia.org/wiki/Collation)).

Applies to text after code snippet as well.

```suggestion
``true`` to ensure values are sorted in numerical order rather than alphabetical order:
```

Reply:
Yes good point, changed this